js/app.js

```diff
@@ -1 +1,50 @@
 require('@uswds/uswds');
+
+const sideNavParents = document.querySelectorAll(".usa-sidenav__item--parent-item");
+const sideNavChild = document.querySelectorAll(".usa-sidenav__sublist--middle-generation");
+const subLists = document.querySelectorAll("ul.usa-sidenav__sublist");
+const middleSubList = document.querySelectorAll(".usa-sidenav__sublist ul");
+
+if (!!anchors) {
```

Comment:
is there a `let anchors` or `var anchors` anywhere? i don't see where this variable is created.  We can maybe cut these 4 lines if so.

Reply:
Anchors is defined in the library.  This is intentional.